### ISSUE TITLE
[ FEATURE ] : Informative feedback to the user searching for a username to connect.

### ISSUE THREAD

**Issue by kshavp** [princekhunt/privateping#13]
**Is your feature request related to a problem? Please describe.**
The feedback from the application that the username does not exist should be informative to the user, thus we can show some "user not found" related feedback instead of what we see in the attached media.

**Describe the solution you'd like**
We can have a toast/popup/alert component to show the feedback.

![image](https://github.com/princekhunt/privateping/assets/95236968/987cf7c3-b32c-4c87-a5e5-4acb800bd9aa)

@princekhunt I'd like to implement this feature under SSOC

**Comment by princekhunt** [comment]
So you'll show toast on each keystroke or on final hit ?

**Comment by princekhunt** [comment]
Button is set to be disabled till a valid username (of friend) entered.

**Comment by kshavp** [comment]
Yes, this is because the button is disabled at the client level, using disabled attribute, which can be easily removed using inspect element and thus this creates a vulnerability.

Thus, for now even if the button is disabled, it is not enough to stop the user. Have a look on the attached media, I have performed a search after removing the disabled attribute manually.

![image](https://github.com/princekhunt/privateping/assets/95236968/55bba729-bbff-4a9d-9cff-bf45580be6bb)

Thus we can possibly avoid this.

**Comment by princekhunt** [comment]
Got it.

**Comment by princekhunt** [comment]
Any update ?

**Comment by kshavp** [comment]
I'm enhancing my current validations as I found something more critical: 
![image](https://github.com/princekhunt/privateping/assets/95236968/0ecd2e6e-3fae-4ae3-a1a9-f75792fb1275)

**Comment by princekhunt** [comment]
Because you're running development server with `DEBUG=True` in settings.
Could you check the same behaviour on production at [https://privateping.bytespot.tech](https://privateping.bytespot.tech) ?

**Comment by kshavp** [comment]
![image](https://github.com/princekhunt/privateping/assets/95236968/230b3f43-3bf0-475e-8ede-cfe722ebd141)
In production i guess its safe.

**Comment by princekhunt** [comment]
Kindly make PR before End of the day, else issue will be open to all to be assigned.
